caravel/templates/moderation/view.html

```diff
@@ -0,0 +1,60 @@
+{% extends "layout.html" %}
```

Comment:
Well, as it stands there's one click for each listing – which is the same as for checkboxes. (It uses Ajax so at least we don't reload the page.) I'm not doing anything fancy vis-a-vis the seller — so that there's no accelerated approval step.